Tasks/Liaskovski/Task1/task1.py

```diff
@@ -0,0 +1,6 @@
+#Task1
+My_deposit = 20000
+Rate = 15
+Term = 5 
+Sum = My_deposit * (1+ Rate / 100 / 12) ** (12 * Term)
+print(Sum , "BYN")
```

Comment:
the Sum value is also related to money so it should be rounded properly before displaying, this is a logical mistake, -15 points